contracts/StablecoinConverter.sol

```diff
@@ -270,26 +276,23 @@ contract StablecoinConverter is EpochTokenLocker {
             updateRemainingOrder(owners[i], orderIds[i], executedSellAmount);
             addBalanceAndBlockWithdrawForThisBatch(owners[i], tokenIdToAddressMap(order.buyToken), executedBuyAmount);
         }
-        // doing all subtractions after all additions (in order to avoid negative values)
+        // Perform all subtractions after additions to avoid negative values
         for (uint256 i = 0; i < owners.length; i++) {
-            (, uint128 executedSellAmount) = getTradedAmounts(volumes[i], orders[owners[i]][orderIds[i]]);
-            subtractBalance(
-                owners[i],
-                tokenIdToAddressMap(orders[owners[i]][orderIds[i]].sellToken),
-                executedSellAmount
-            );
+            Order memory order = orders[owners[i]][orderIds[i]];
```

Comment:
I think there was a gas efficiency reason for not doing this? @josojo do you remember?

Reply:
Ben's new solution is loading one more time the orders[owners[i]][orderIds[i]] into memory, I suppose.

But since we made already so many compromises with gas, I think, it is okay :)
And Ben's solution improves the readability for sure.

Reply:
using the order more than one in this location was one of the reasons for the change. Also I was stuck with a line width versus number of lines per function battle during this PR. Had to make some decisions.

Turns out that setting the `order` allowed for fewer line breaks and I managed to keep within the 57 lines per function.